src/app/hearing-part/components/listing-note-list/listing-note-list.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
+import { NoteListComponent } from '../../../notes/components/notes-list/note-list.component';
+import { Note } from '../../../notes/models/note.model';
+import { ListingCreateNotesConfiguration } from '../../models/listing-create-notes-configuration.model';
+import { getNoteViewModel, NoteViewmodel } from '../../../notes/models/note.viewmodel';
+import { NotesPreparerService } from '../../../notes/services/notes-preparer.service';
+
+@Component({
+  selector: 'app-listing-note-list',
+  templateUrl: './listing-note-list.component.html',
+  styleUrls: ['./listing-note-list.component.scss']
+})
+export class ListingNoteListComponent implements OnInit {
+  @ViewChild('notesList') noteList: NoteListComponent;
+  @ViewChild('freeTextNotesList') freeTextNoteList: NoteListComponent;
+
+  public _notes: Note[];
```

Comment:
variables with `_` should be used as private. Can you add private to it and make getter?